attributes/default.rb

```diff
@@ -90,4 +90,9 @@
 default['cloud']['queue_config']['capacity']             = "10000"
 default['cloud']['queue_config']['max_sample_per_send']  = "5000"
 default['cloud']['queue_config']['batch_send_deadline']  = "60s"
-default['cloud']['metrics']['port']                      = "9096"
\ No newline at end of file
+default['cloud']['metrics']['port']                      = "9096"
+
+default['hopsmonitor']['kube_ca'] = "#{node['certs']['dir']}/kube/kube-ca.cert.pem"
```

Comment:
Small thing, but please use spaces to align all of the   = "...." in the file.
See lines 90-93 for example.